lib/export_vacancy_records_to_big_query.rb

```diff
@@ -0,0 +1,118 @@
+require 'google/cloud/bigquery'
+
+class ExportVacancyRecordsToBigQuery
+  attr_reader :dataset
+
+  def initialize(bigquery: Google::Cloud::Bigquery.new)
+    @dataset = bigquery.dataset ENV.fetch('BIG_QUERY_DATASET')
+  end
+
+  # rubocop:disable Metrics/AbcSize
```

Comment:
Are we really getting any value from this rubocop metric? It feels like in the codebase we're just disabling and enabling it for the sake of dismissing it